### ISSUE TITLE
Where are the Windows PowerShell communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/powershell#63]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is Windows PowerShell supported by one or more large organizations?
* Does Windows PowerShell have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about Windows PowerShell? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for Windows PowerShell? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in Windows PowerShell? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in Windows PowerShell?
* Are there popular newsletters for Windows PowerShell?
* Is Windows PowerShell taught at programming bootcamps? (If so, what are some examples?)
* Is Windows PowerShell taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about Windows PowerShell?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by kchenery** [comment]
Answering some of these points....

* PowerShell is supported by Microsoft.  See https://docs.microsoft.com/en-us/powershell/scripting/powershell-scripting?view=powershell-6
* Jeffrey Snover is the original author and champion of PowerShell and is heavily involved in its development.  He is probably the closest to a "community manager" for now.  Jeffrey's twitter handle is: https://twitter.com/jsnover
* There are several community initiatives listed here: https://docs.microsoft.com/en-us/powershell/#pivot=main&panel=community

**Comment by kytrinyx** [comment]
Fantastic, thank you!

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌼

**Comment by lavanyasreepada** [comment]
Hi 

Thank you Good Information 
Please go through this, you can attend a free demo here [PowerShell Training](https://mindmajix.com/powershell-training) 

Thanks in advance, 
Lavanya Sreepada
